mediaproviders/src/main/java/com/kaltura/playkit/providers/ovp/OVPMediaAsset.java

```diff
@@ -53,6 +51,12 @@ public Boolean getRedirectFromEntryId() {
     }
 
     public String getUUID() {
-        return  !TextUtils.isEmpty(getEntryId()) ? getEntryId() : getReferenceId();
+
+        if (gson == null) {
```

Comment:
we should ignore KS also here

Reply:
this is happening in the gson level

Reply:
right, I missed it and thought it is only for OTT